streamer/src/quic.rs

```diff
@@ -506,109 +263,24 @@ pub fn spawn_server(
     max_unstaked_connections: usize,
     stats: Arc<StreamStats>,
 ) -> Result<thread::JoinHandle<()>, QuicServerError> {
-    let (config, _cert) = configure_server(keypair, gossip_host)?;
-
     let runtime = rt();
-    let (_, mut incoming) = {
+    let task = {
         let _guard = runtime.enter();
-        Endpoint::new(EndpointConfig::default(), Some(config), sock)
-            .map_err(|_e| QuicServerError::EndpointFailed)?
-    };
-
+        crate::nonblocking::quic::spawn_server(
```

Comment:
`configure_server` is called inside of `nonblocking::quic::spawn_server`, so it's performed inside of the runtime now.  Does this cause a problem?

Reply:
Should be fine, I think